.spectral.yaml

```diff
@@ -0,0 +1,8 @@
+documentationUrl: https://github.com/italia/api-oas-checker-rules
+extends:
+  [
+    "https://github.com/italia/api-oas-checker-rules/releases/download/1.1/spectral-modi.yml",
+  ]
+rules:
+  servers-use-https: off
+  paths-status: off # Enable this once authorization server migration is completes and we can remove the /authorization-server prefix
```

Comment:
We still need to clarify if we can remove this now that auth server migration is complete, maybe you can ask on Slack?